src/assets.rs

```diff
@@ -401,11 +401,21 @@ fn asset_from_cache<T: serde::de::DeserializeOwned>(
 
 #[cfg(target_os = "macos")]
 fn macos_dark_mode_active() -> bool {
-    let mut defaults_cmd = std::process::Command::new("defaults");
-    defaults_cmd.args(&["read", "-globalDomain", "AppleInterfaceStyle"]);
-    match defaults_cmd.output() {
-        Ok(output) => output.stdout == b"Dark\n",
-        Err(_) => true,
+    const PREFERNECES_FILE: &str = "Library/Preferences/.GlobalPreferences.plist";
```

Comment:
Is this always the path? Regardless of Mac OS version and regardless of system/user settings?

Reply:
> Regardless of mac OS version

Yes, to the best of what I know. Its been at this path since at least 10.7 in 2011 (which is the lowest Rust supports) but probably even before that so I'm confident this will stick around.

> regardless of system/user settings?

I think so? I don't have any documentation to back it up but I have never seen it anywhere else different. If this isn't enough, then we should just use CFPreferences so this is abstracted away.